data/language/en-GB.txt

```diff
@@ -3668,6 +3668,12 @@ STR_6346    :Tile Inspector: Decrease element height
 STR_6347    :Path additions cannot be placed on level crossings!
 STR_6348    :Remove level crossing first!
 STR_6349    :Random title sequence
+STR_6350    :Scatter
+STR_6351    :Scenery Scatter Tool
+STR_6352    :Density
+STR_6353    :{SMALLFONT}{BLACK}Low density (25%)
+STR_6354    :{SMALLFONT}{BLACK}Medium density (50%)
+STR_6355    :{SMALLFONT}{BLACK}High density (75%)
```

Comment:
These lines should probably more closely reflect what is actually happening, e.g. `1x`, `2x`, `3x`. (Right now, `25%` could be interpreted as 16 * 16 * 0.25 = 64, while in reality only 16 elements are placed.)

Reply:
Just get rid of the percentages and title them `Low density`, etc. The 1x/2x suffices don't add anything informative.